modules/rosa-sts-aws-requirements.adoc

```diff
@@ -21,6 +21,102 @@ The following prerequisites must be complete before you deploy a {product-title}
 Customers are encouraged, but not mandated, to deploy resources in a Virtual Private Cloud (VPC) separate from the VPC hosting {product-title} and other Red Hat supported services.
 ====
 
+//
+// Start work for OSDOCS-3198
+//
+[id="rosa-associating-account_{context}"]
+== Associating your AWS account
+
+To perform {product-title} (ROSA) cluster provisioning tasks in {cluster-manager}, you must create `ocm-role` and `user-role` IAM resources in your AWS account and link them to your Red Hat organization.
+
+The `ocm-role` ARN is stored as a label in your Red Hat organization while the `user-role` ARN is stored as a label inside your Red Hat user account. Red Hat uses these ARN labels to confirm that the user is a valid account holder and that {cluster-manager} has the correct permissions to perform the necessary checks for this user account.
+
```

Comment:
@arendej and @0kashi I tried to incorporate my understanding of @vkareh's explanation of why we require linking. Is this useful, and did I manage to capture the concept accurately?